Autoscaler panics if prometheus doesn't return data

If a Prometheus query returns empty, the Autoscaler will panic with the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x19 pc=0x1d7b5e3]

goroutine 110 [running]:
github.com/hashicorp/nomad-autoscaler/policyeval.(*BaseWorker).handlePolicy(0xc00039f3b0, 0x25fe5e0, 0xc000392740, 0xc000f92050, 0x0, 0x0)
        /home/circleci/project/project/policyeval/base_worker.go:165 +0xac3
github.com/hashicorp/nomad-autoscaler/policyeval.(*BaseWorker).Run(0xc00039f3b0, 0x25fe5e0, 0xc000392740)
        /home/circleci/project/project/policyeval/base_worker.go:77 +0x2ed
created by github.com/hashicorp/nomad-autoscaler/agent.(*Agent).initWorkers
        /home/circleci/project/project/agent/agent.go:108 +0x1d6
```

--- a/policyeval/base_worker.go
+++ b/policyeval/base_worker.go
@@ -162,7 +162,7 @@ func (w *BaseWorker) handlePolicy(ctx context.Context, eval *sdk.ScalingEvaluati
 	// tracking how long it takes to run all the checks within a policy.
 	metrics.MeasureSinceWithLabels([]string{"scale", "evaluate_ms"}, evalStartTime, labels)
 
-	if winningHandler == nil || winningAction.Direction == sdk.ScaleDirectionNone {
+	if winningHandler == nil || winningAction == nil || winningAction.Direction == sdk.ScaleDirectionNone {
 		logger.Debug("no checks need to be executed")
 		return nil
 	}

